.gitignore

```diff
@@ -62,4 +62,5 @@ target/
 *.sublime-workspace
 
 # Picasa stuff
-*.ini
\ No newline at end of file
+*.ini
+content/maxwell1_fundamentals/fundamental_laws/images/~$BC.pptx
```

Comment:
are you guys using this file to generate figures? If so, it would be great to include it in the repo so that the figures can be reproduced or adapted to be applicable to other elements of the site